src/EmailManager/InvoiceEmailManager.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\EmailManager;
+
+use Knp\Bundle\SnappyBundle\Snappy\Response\PdfResponse;
+use Sylius\Component\Mailer\Sender\SenderInterface;
+use Sylius\InvoicingPlugin\Entity\InvoiceInterface;
+
+final class InvoiceEmailManager implements InvoiceEmailManagerInterface
```

Comment:
Since all similar classes in Sylius are `Managers` i wanted to stick to this pattern in order to be consistent. It's not the best name for it, however. 

It will be changed in review fixes commit